backend/src/schema/resolvers/notifications.js

```diff
@@ -41,13 +40,15 @@ export default {
         default:
           orderByClause = ''
       }
-
+      const offset = args.offset ? `SKIP ${args.offset}` : ''
+      const limit = args.first ? `LIMIT ${args.first}` : ''
```

Comment:
Our GraphQL schema saves us here, because you must enter an Integer. But if that wouldn't be the case, we would have a cypher injection vulnerability here.